source_manager.go

```diff
@@ -27,6 +27,18 @@ var sanitizer = strings.NewReplacer(":", "-", "/", "-", "+", "-")
 // sufficient for any purpose. It provides some additional semantics around the
 // methods defined here.
 type SourceManager interface {
+	// SourceManager adds a Release method to the sourceDeducer method set. The
+	// sourceDeducer method set is defined in a separate interface because it's
+	// used by the sourceBridge interface as well, but sourceBridges don't get a
+	// Release method.
+	sourceDeducer
```

Comment:
I was wondering to myself if this was the right way to do it while typing my earlier comment. If we do this, then the `SourceManager` interface in godoc will no longer have all those methods visible, right?